### ISSUE TITLE
Fix package details in PyPi and `setup.py`

### ISSUE THREAD

**Issue by lvaylet** [google/slo-generator#262]
At the time of writing, the PyPi package is still in `Alpha` and the supported Python versions are not consistent (see also #257):

![image](https://user-images.githubusercontent.com/10340090/196389376-7037417d-5a00-4eea-b6ac-e6de89501fe4.png)

Update these details for the next release.

Update the author email too (currently `ocervello@google.com`) so users do not get an OoO reply?

**Comment by ocervell** [comment]
You can `,` separate authors, just replace by `email1, email2`, or even use the format `first_name last_name <email1>, first_name last_name <email2>`

**Comment by lvaylet** [comment]
Fixed with #267. I ended up with these fields in `setup.cfg` to give credit where credit is due:

```cfg
author = Google Inc.
author_email = olivier.cervello@gmail.com
maintainer = Laurent VAYLET
maintainer_email = laurent.vaylet@gmail.com
```
